pkg/controller/aci_istio.go

```diff
@@ -289,12 +273,22 @@ func (t *AciIstioOperatorHandler) ObjectCreated(obj interface{}) {
 	if err != nil {
 		log.Error(err.Error())
 	}
+
 }
 
 func (t *AciIstioOperatorHandler) ObjectDeleted(obj interface{}) {
-	log.Info("OperatorHandler.ObjectDeleted")
+
+	log.Info("Deleting istio-system namespace")
+	cmd := exec.Command("kubectl", "delete", "ns", "istio-system",  "--grace-period=0", "--force")
+	out, err := cmd.Output()
+	if err != nil {
+		log.Info("Failed:",err)
```

Comment:
Can you return the code form here otherwise both success and Failed message will be seen

Reply:
Fixed.